src/App.js

```diff
@@ -1,9 +1,13 @@
-import Calculator from './components/calculator';
+import './App.css';
+import React from 'react';
+import Calculator from './components/MyCalculator';
 
-export default function MyApp() {
+function App() {
   return (
-    <div className="mul">
+    <div className="App">
       <Calculator />
     </div>
   );
 }
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/236306114-9a114da0-f9d8-494f-9f79-ed01fef2dabf.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message more meaningful to make your pull request professional :+1: